CHANGELOG.md

```diff
@@ -0,0 +1,268 @@
+    ```
+    
+- Rework code and profiles for refactored SDK ([#214](https://github.com/edgexfoundry/app-service-configurable/issues/214)) ([#d1338e8](https://github.com/edgexfoundry/app-service-configurable/commits/d1338e8))
+
+    ```
+    BREAKING CHANGE:
+    SetOutputData function has changed to SetResponseData
+    ```
+
+- Consolidate function pipeline configuration ([#208](https://github.com/edgexfoundry/app-service-configurable/issues/208)) ([#d52bde6](https://github.com/edgexfoundry/app-service-configurable/commits/d52bde6))
+
+    ```
+    BREAKING CHANGE:
+    - TransformToXML & TransformToJSON functions have be combined into the Transform function with the new 'Type' parameter
+    - CompressWithGZIP & CompressWithZLIB functions have be combined into the Compress function with the new 'Algorithm ' parameter
+    - HTTPPost, HTTPPut, HTTPPostJSON, HTTPPutJSON, HTTPPostXML and HTTPPutXML have be combined into the HTTPExport function with the new 'method' parameter and utilizing the existing 'MimeType' parameter
+    - BatchByCount, BatchByTime, BatchByTimeAndCount have be combined into the Batch function with the new 'Mode' parameter
+    ```
+
+- Update profiles for change in Trigger configuration ([#203](https://github.com/edgexfoundry/app-service-configurable/issues/203)) ([#ca2daa4](https://github.com/edgexfoundry/app-service-configurable/commits/ca2daa4))
+
+    ```
+    BREAKING CHANGE:
+    - [Binding] section has been renamed to [Trigger]
+    - [MessageBus] section has been renamed to [EdgexMessageBus] and moved under [Trigger]
+    - [MqttBroker] section has been renamed to [ExternalMqtt] and moved under [Trigger]
+    ```
+
+- Update profiles for refactored http export with secret header ([#198](https://github.com/edgexfoundry/app-service-configurable/issues/198)) ([#6a27a6a](https://github.com/edgexfoundry/app-service-configurable/commits/6a27a6a))
+    ```
+    BREAKING CHANGE:
+    Parameters have changed for HTTP Post/Put with SecretHeader
+    ```
+    
+- Renaming blackbox-tests profile to functional-tests ([#179](https://github.com/edgexfoundry/app-service-configurable/issues/179)) ([#007494f](https://github.com/edgexfoundry/app-service-configurable/commits/007494f))
+
+- Remove SecretStoreExclusive configuration ([#158](https://github.com/edgexfoundry/app-service-configurable/issues/158)) ([#45b32cc](https://github.com/edgexfoundry/app-service-configurable/commits/45b32cc))
+
+    ```
+    BREAKING CHANGE:
+    SecretStoreExclusive no longer used
+    ```
+
+- Remove use of deprecated MQTTSend ([#145](https://github.com/edgexfoundry/app-service-configurable/issues/145)) ([#86d88ba](https://github.com/edgexfoundry/app-service-configurable/commits/86d88ba))
+
+    ```
+    BREAKING CHANGE:
+    MQTTSend no longer available. Replaced by MQTTSecretSend 
+    ```
+### Documentation 📖
+- Add badges to readme ([#aa00b56](https://github.com/edgexfoundry/app-service-configurable/commits/aa00b56))
+### Build 👷
```

Comment:
Please insert new lines before and after title- easier to read the separation boundary and also lint from markdown